icon_validator/rules/workflow_validators/workflow_files_validator.py

```diff
@@ -0,0 +1,18 @@
+import os
+
+from icon_validator.rules.validator import KomandPluginValidator
+from icon_validator.exceptions import ValidationException
+
+
+class WorkflowFilesValidator(KomandPluginValidator):
+
+    def validate(self, spec):
+        d = spec.directory
+
+        if not os.path.isfile("{}/{}".format(d, "workflow.spec.yaml")):
+            raise ValidationException("File workflow.spec.yaml does not exist in: ", d)
+        if not os.path.isfile("{}/{}".format(d, "help.md")):
+            raise ValidationException("File help.md does not exist in: ", d)
+        if not os.path.isfile("{}/{}".format(d, "extension.png")):
+            raise ValidationException("File extension.png does not exist in: ", d)
+        # TODO check for .icon file
```

Comment:
add check for .icon file

Reply:
I will do this in the next validator release. I want to get this PR out so we have some validators and so the next one will be smaller and easier to read.